docs/TargetMaintenance/CreatingAUnifiedTarget.md

```diff
@@ -37,12 +37,47 @@ Betaflight advises target creators against adding any setting to their Unified T
 
 All of these are really user preferences, and adding them as part of a Unified Target Configuration just means that most customers of this hardware will have to spend extra time on reverting the presets before they can set up their own preferences. There are no strict rules here, and what is reasonable will be dependent on what type of target it is, but manufacturers are advised to use common sense.
 
+## 2. Creating a Unified Target Configuration for a new firmware
 
-## 2. Creating a Unified Target Configuration
+These instructions explain how to create a new Unified Target configuration without an existing Betaflight target. If your target already exists in non-unified format please skip to "Creatings a Unified Target Configuration for an existing firmware" 
+Creating Unified Targets for a new board means you must start with a generic firmware for your MCU and build up the required pin mappings. Start with a generic firmware and then validate pin mappings and hardware functionality.
+
+### 2.1 Obtaining a unified target firmware for your MCU
+
+Betaflight Configuratior distributes customised targets featuring settings specific to each board. To create firmware for a new board it is recommended to build a generic firmware for your MCU type - STM32F411, STM32F405, STM32F7X2, and so on.
+These firmware types can be created by building locally and specifying the MCU type such as `make STM32F05`. Firmware files are created in the `obj` directory and can be flashed by choosing `Load local file` in Betaflight Configurator.
+
+### 2.2 Flashing the unified target firmware
+
+(Theoretically there is no need for the board to match the firmware that is flashed in this step, but there is a chance that the board configuration is setting an input pin on the board to be an output pin, thus leading to a short and potential hardware damage.)
+
+- the important part is that this is the firmware for the MCU type your board uses;
+
+- make sure to enable 'Full chip erase' before flashing, or reset to default with `defaults` in CLI;
```

Comment:
```suggestion
- the important part is that this is the firmware for the MCU type your board uses.

- make sure to enable 'Full chip erase' before flashing, or reset to default with `defaults` in CLI.
```

Reply:
Replaced semicolon with full stop here and also in the paragraph below which I took this pattern from.